example_encryted_file/lib/main.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:io';
+import 'dart:typed_data';
+import 'package:http/http.dart' as http;
+
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+import 'package:path_provider/path_provider.dart';
+
+void main() async {
```

Comment:
why not do it in `test` folder?

Reply:
you have to add `path_provider` dependency and also fake image data, not real data.

Reply:
<img width="745" alt="image" src="https://github.com/linagora/matrix-dart-sdk/assets/6462404/c243d132-f391-4c00-bbcd-695d4a37b304">